brave-jaxrs2/pom.xml

```diff
@@ -37,6 +37,11 @@
             <version>1</version>
         </dependency>
         <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.2</version>
```

Comment:
or if it has nothing to do with any of the dependencies here, ignore my comment :P